react/config-overrides.js.template

```diff
@@ -18,8 +18,14 @@ module.exports = function override(config, env) {
       ignoreFile: '.sentrycliignore',
       ignore: ['webpack.config.js'],
       configFile: 'sentry.properties',
-      reactComponentAnnotation: {enabled:true},
+    })
+  );
+  config.plugins.push(
+    codecovWebpackPlugin({
+      enableBundleAnalysis: process.env.CODECOV_TOKEN !== undefined,
```

Comment:
ideally we'd have some other condition on the environment here (e.g. `process.env.CI` `process.env.REACT_APP_ENVIRONMENT`) otherwise this can mask configuration issues (CODECOV_TOKEN not set by mistake)

maybe something like this so you can test it locally if you want to by setting the token?
```
process.env.CI !== undefined || process.env.CODECOV_TOKEN !== undefined
```